packaging/building-a-package/en.md

```diff
@@ -61,9 +61,8 @@ Your folder should look something similar to this:
 ```
 | common/
 | your-package/
-| - Makefile
+| Makefile
 | Makefile.common
-| Makefile.toplevel
```

Comment:
Should probably just change this to Makefile instead and have it be above Makefile.common, to illustrate that there should be a Makefile in your "repo" directory.